### PR TITLE
moving insights templates down, getting started up

### PR DESCRIPTION
**Pull request by yves-chan**

### CODE REVIEW

File: Workbooks/Azure Monitor - Getting Started/categoryResources.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "en-us": {"name":"Getting started with workbooks", "description": " ", "order": 2000}
+    "en-us": {"name":"Getting started with workbooks", "description": " ", "order": 150}
```
Comment (yves-chan) [inline Workbooks/Azure Monitor - Getting Started/categoryResources.json]: I based this number off of the Virtual Machines order which is 200

**Comment by gardnerjr** [comment]
add a screenshot maybe of what the gallery looks like after?

**Comment by yves-chan** [comment]
@gardnerjr yes I can do that. I assume there isn't a way to preview what the gallery would look like?

**Comment by gardnerjr** [comment]
no, currently no good way to do that, since any change could affect *lots* of galleries.  maybe the CI build should output a giant tree view of what's where?  not sure there'd be an easy way to spot the differences though?

**Pull request merged**